dbos/_dbos_config.py

```diff
@@ -96,6 +98,8 @@ def parse_database_url_to_dbconfig(database_url: str) -> DatabaseConfig:
 class OTLPExporterConfig(TypedDict, total=False):
     logsEndpoint: Optional[str]
     tracesEndpoint: Optional[str]
+    mergedLogsEndpoints: Optional[List[str]]
+    mergedTracesEndpoints: Optional[List[str]]
 
```

Comment:
This is a little tricky. If we take out the `logsEndpoint` and `tracesEndpoint` (top 2) or if we try to make them plural, then our JSON file parsing logic breaks. 

So our logic is to merge everything into the "merged" bits during ... well... the merged phase. LMK if you think another way is better

Reply:
It should be possible to make the existing fields plural. You have to add some special-casing in `load_config` to turn the loaded string into a list.

Reply:
Roger. This should be in place now, have a look @kraftp